### ISSUE TITLE
When submitting 'search' author button has 'add author' functionality

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4306]
### The Problem

Backoffice. Submit an article. 

<img width="885" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/60051f9c-0da7-45cd-86af-c165361f83f5">

When Clicking "search" if there is a match, that author is automatically added to the article.  This is not expected behaviour, even though it is described as such in the text above, the button stands out more, and is clearly at odds with that text.

### What I'd like to see:

- Button text changed to 'Search and Add' would prevent confusion.

**Comment by ajrbyers** [comment]
@StephDriver at the risk of being pedantic "Search and Add" doesn't cover the posibility that a user isn't found so could also compound confusion, no?

**Comment by StephDriver** [comment]
> [@StephDriver](https://github.com/StephDriver) at the risk of being pedantic "Search and Add" doesn't cover the posibility that a user isn't found so could also compound confusion, no?

Agreed. Personally I'd prefer this be two steps, first search. Then add, so the user knows the result of the search before adding it.

But that ends up being quite clunky. One way or another I do think this user-interaction needs to be rethought.  Right now it is confusing.  But I agree, my proposed change doesn't eliminate all of that confusion.

**Comment by StephDriver** [comment]
Assigning myself to this to give it a bit more thought (needs investigation) ahead of it being discussed in Backlog Refinement.

**Comment by StephDriver** [comment]
Closed as is now a duplicate, since this work has been incorporated in redesign of that view in 
- #4519
